lib/philomena_web/topic_json.ex

```diff
@@ -0,0 +1,34 @@
+defmodule PhilomenaWeb.TopicJson do
+  alias PhilomenaWeb.UserAttributionView
+
+  def as_json(%{hidden_from_users: true} = topic) do
+    %{
+      id: topic.id,
+      slug: nil,
+      title: nil,
+      post_count: nil,
+      view_count: nil,
+      sticky: nil,
+      last_replied_to_at: nil,
+      locked: nil,
+      user_id: nil,
+      author: nil
+    }
+  end
```

Comment:
Style: function clauses of the same name and arity should have no blank line between them